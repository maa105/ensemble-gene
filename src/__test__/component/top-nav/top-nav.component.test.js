import React from 'react';
import TopNav from '../../../components/top-nav/top-nav.component';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

test('Top nav renders as expected', () => {
  const component = renderer.create(
    <Router>
      <TopNav />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
