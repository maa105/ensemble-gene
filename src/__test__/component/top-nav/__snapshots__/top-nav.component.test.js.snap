// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top nav renders as expected 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="https://www.ensembl.org"
      >
        Ensembl
      </a>
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
    </div>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
    >
      <ul
        className="nav navbar-nav"
        role={null}
      >
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="/find-proteins"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Find Protein
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="/hgvs"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            HGVS
          </a>
        </li>
      </ul>
      <ul
        className="nav navbar-nav navbar-right"
        role={null}
      >
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="https://rest.ensembl.org/"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            API
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
